### PR TITLE
Update description of the language.

### PR DESCRIPTION
**Pull request by hepin1989**
More about Scala's business object.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: frontpage
 
 # Header texts
 headerTitle: "The Scala Programming Language"
-headerSubtitle: "Scala combines object-oriented and functional programming in one concise, high-level language. Scala's static types help avoid bugs in complex applications, and its JVM and JavaScript runtimes let you build high-performance systems with easy access to huge ecosystems of libraries."
+headerSubtitle: "Scala is a **consistent**, **elegant** and **expressive** language. Scala helps you to build highly **elastic**, **resilient** and **responsive** applications quickly with huge ecosystems, expressive language features and great inter-operation, run them fast on JVM and JavaScript runtimes,"
```
Comment (hepin1989) [inline index.md]: Maybe:
"Scala is a **consistent**, **elegant** and **expressive** language, combines bests of functional and object-oriented programming into one.

File: index.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: frontpage
 
 # Header texts
 headerTitle: "The Scala Programming Language"
-headerSubtitle: "Scala combines object-oriented and functional programming in one concise, high-level language. Scala's static types help avoid bugs in complex applications, and its JVM and JavaScript runtimes let you build high-performance systems with easy access to huge ecosystems of libraries."
+headerSubtitle: "Scala is a **consistent**, **elegant** and **expressive** language, combines bests of functional and object-oriented programming. Scala helps you to build highly **elastic**, **resilient** and **responsive** applications quickly with huge ecosystems, expressive language features and great inter-operation, run them fast on JVM and JavaScript runtimes,"
```
Comment (hepin1989) [inline index.md]: Maybe:
Scala is a **consistent**, **elegant** and **expressive** static typed language, combines bests of functional and object-oriented programming

**Comment by hepin1989** [comment]
image

image

image

image

image

**Comment by sjrd** [comment]
That text has been the topic of many discussions over the years, with relatively recent changes (in March) for example at #852 and #854. You'll want to make a pretty strong case why your proposed formulation is better than the previous one and how it addresses the concerns of the many previous comments.

**Comment by zhongdj** [comment]
At least DO NOT make me unemployed.

**Comment by propensive** [comment]
Just to nudge this PR a little, @hepin1989, as @sjrd says, this text has always been *difficult* to change because it's some of the most important marketing for Scala, so I think there's a general unwillingness to merge changes unless there is a clear improvement, and agreement that it's definitely an improvement from the various stakeholders in Scala.

So I don't want to discourage you or anyone else from suggesting improvements to this text, because it *is* important, but there would need to be lots of support for any change. I don't feel confident enough merging this without more feedback from more people.

Ridiculous as this may sound, one approach may be to split this already-tiny PR into even smaller, incremental PRs than can be judged in isolation. It might also be necessary to encourage some other people in the community to share their comments.

So, I'm going to close *this* PR, but I don't believe it should be the end of the road for the changes you've proposed. They just need to be made in a way which is more amenable to being merged.

**Pull request closed without merging**